internal/pkg/template/templates/environment/partials/cfn-execution-role.yml

```diff
@@ -5,9 +5,6 @@ CloudformationExecutionRole:
     'aws:copilot:description': 'An IAM Role for AWS CloudFormation to manage resources'
   DeletionPolicy: Retain
   Type: AWS::IAM::Role
-{{- if not .VPCConfig.Imported}}
-  DependsOn: VPC
-{{- end}}
```

Comment:
I can't remember why we need this dependsOn. Is there anything to worry about if we remove it 🤔

Reply:
I've discussed with several founders 🤣 and the decision was to remove it cuz there is no reason to keep it if no one knows why it's there.